src/ui/qgsfeatureselectiondlg.ui

```diff
@@ -11,7 +11,7 @@
    </rect>
   </property>
   <property name="windowTitle">
-   <string>Dialog</string>
+   <string>Link existing child features</string>
```

Comment:
I think it would be worth adding the (child) layer name in here, is that possible?

Reply:
You mean the parent layer name for selecting its childs ? 

Done 78712dd. I add a method in QgsVectorLayerUtils to get feature display string because it's the second time I had to do this.